### PR TITLE
[nt] Change example placeholder to editable text

### PR DESCRIPTION
**Pull request by nathaniel-mage**
# Summary

### Change the custom code placeholder to be default value
- Removed placeholder
- Update `CODE_EXAMPLE` to be more instructional

# Tests

Localhost and ran some queries.
### Queries
image

### Default values (editable)
Screen Shot 2022-06-08 at 12 58 38 PM

_You can tell it's a value because it's colored_

cc: @johnson-mage 

Is this what you were thinking of @tommydangerous? Or did you want to display it somewhere else?

### CODE REVIEW

**Comment by nathaniel-mage** [comment]
Found another bug, will fix don't review yet.

**Comment by nathaniel-mage** [comment]
### Fixed Filter placeholder and display bug
image

**Comment by nathaniel-mage** [comment]
And it still works on custom actions
image

Additionally, if you want to add your own default value you can include it.

File: mage_ai/frontend/components/ActionForm/actions.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ const columns: {
       Add @transformer_action, then begin typing Python code inside a function. \
       You may refer to the current dataset as df.',
     code: {
+      default: CODE_EXAMPLE,
```
Comment (johnson-mage) [inline mage_ai/frontend/components/ActionForm/actions.ts:28]: if this `custom` property is the exact same in columns and rows constants, let's create a separate variable for it and  use it in both places. It's a hassle to have to update the code in two places every time we make a change.

**Review by johnson-mage — APPROVED** [review]
approved after addressing comment

File: mage_ai/frontend/components/ActionForm/actions.ts (hunk 1 above)
Comment (nathaniel-mage) [reply mage_ai/frontend/components/ActionForm/actions.ts:28]: That's what code example is for, or am i misunderstanding what you mean?

Comment (johnson-mage) [reply mage_ai/frontend/components/ActionForm/actions.ts:28]: meant the whole `custom` prop: 
```
custom: {
    title: 'Custom code',
    description: 'Write your own custom cleaning function. \
      Add @transformer_action, then begin typing Python code inside a function. \
      You may refer to the current dataset as df.',
    code: {
      default: CODE_EXAMPLE,
      multiline: true,
      values: USER_INPUT,
    },
```

Comment (nathaniel-mage) [reply mage_ai/frontend/components/ActionForm/actions.ts:28]: One is for the dataset page and the other is for column specific page

Comment (nathaniel-mage) [reply mage_ai/frontend/components/ActionForm/actions.ts:28]: Should I write different copy for them then?

Comment (nathaniel-mage) [reply mage_ai/frontend/components/ActionForm/actions.ts:28]: This is the logic that selects which to look at: (axis === AxisEnum.ROW ? actions.rows : actions.columns)?.[actionType];
Adding a third variable might break it.

**Comment by nathaniel-mage** [comment]
Merged but adding in the enhancement. Didn't include this in merge due to type errors resulting in failed build.

**Pull request merged**